sw/device/tests/crypto/cryptotest/firmware/BUILD

```diff
@@ -207,6 +207,24 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "extclk_sca_fi",
+    srcs = ["extclk_sca_fi.c"],
+    hdrs = [
+        "extclk_sca_fi.h",
+        "status.h",
```

Comment:
I'm not sure this belongs as a header, shouldn't this be a dep? Though I also think that this additional status.h file is unneeded too (see below comment on just using `TRY()`). (Though please correct me if I am misunderstanding something here.)

Reply:
Removed, thanks.